tests/test_req_vars.py

```diff
@@ -107,6 +108,18 @@ def test_reconstruct_url(self):
 
         self.assertEqual(expected_uri, req.uri)
 
+    def test_nonlatin_path(self):
+        if not six.PY3:
+            self.skipTest('Test only applies to Python 3')
+
+        cyrillic_path = six.u('/hello_\u043f\u0440\u0438\u0432\u0435\u0442' )
```

Comment:
Nit: No need to use `six.u`, since Falcon only supports versions of Python that recognize the literal `u` prefix.